Orange/widgets/visualize/owscatterplot.py

```diff
@@ -537,6 +586,9 @@ def migrate_settings(cls, settings, version):
                 settings["auto_commit"] = settings["auto_send_selection"]
             if "selection_group" in settings:
                 settings["selection"] = settings["selection_group"]
+        if version < 5:
```

Comment:
Thanks, @VesnaT, but I think this is too strong: `jitter_continous` was a valid parameter somewhere before 5. Shouldn't we only set it to True if it is not in the dictionary?